ParentalLockRemover.te

```diff
@@ -0,0 +1,53 @@
+printf("Remove Parental Lock Kronos2308 -.-")
```

Comment:
Please remove your name from the script, for uniformity across the repository. You may however, add a line with # in the front (python style comment) with your name, if you wish (even though most of the code here is taken from the syswipe script it seems)